kitsune/wiki/views.py

```diff
@@ -520,6 +520,21 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get latest revision which is still not approved. Use "latest_revision.id" to get the id.
+    try:
+        latest_revision = Revision.objects.filter(
+            document=doc,
+            is_approved=False).order_by('-created')[0]
+        latest_revision_id = latest_revision.id
+    except IndexError:
+        latest_revision_id = None
+
+    # Get Current Revision if "only" there is any approved revision.
```

Comment:
1. Don't use quotes (`"`) for emphasis. It isn't correct. Use asterisks (`*`).
2. This comment just restates the code below. That isn't very useful.